Formula/bifrost.rb

```diff
@@ -1,16 +1,15 @@
 class Bifrost < Formula
   # cite Holley_2019: "https://doi.org/10.1101/695338"
-  desc "Highly parallel build/index of coloured/compacted de Bruijn graphs"
+  desc "Construction, indexing and querying of colored/compacted de Bruijn graphs"
   homepage "https://github.com/pmelsted/bifrost"
-  url "https://github.com/pmelsted/bifrost/archive/v1.0.4.tar.gz"
-  sha256 "94ac84861c36bc3fb9ea7b4a630ce391c79236289e5fab7aa627dc07b286f013"
+  url "https://github.com/pmelsted/bifrost/archive/v1.0.5.tar.gz"
```

Comment:
Please update the `sha256`. Running `brew fetch -s bifrost` will tell you the new SHA-256.